src/main/java/gregtech/api/metatileentity/MetaTileEntityHolder.java

```diff
@@ -320,9 +320,12 @@ public BlockPos pos() {
         return getPos();
     }
 
+    @SuppressWarnings("all") // yes it CAN actually be null
```

Comment:
More specific warning suppression would be better here. Otherwise this may lead to future things being missed later down the line.